_ml-commons-plugin/remote-models/guardrails.md

```diff
@@ -636,4 +636,5 @@ OpenSearch responds with an error.
 
 ## Next steps
 
-- For more information about configuring guardrails, see [The `guardrails` parameter]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/model-apis/register-model/#the-guardrails-parameter).
\ No newline at end of file
+- For more information about configuring guardrails, see [The `guardrails` parameter]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/model-apis/register-model/#the-guardrails-parameter).
+- For a guardrails tutorial, see [Using Amazon Bedrock guardrails]({{site.url}}{{site.baseurl}}/ml-commons-plugin/tutorials/bedrock-guardrails/)
```

Comment:
"For a guardrails tutorial" => "For a tutorial demonstrating how to use Amazon Bedrock guardrails"?